examples/passive_scalar/benchmark/gaussian2d/index.md

```diff
@@ -0,0 +1,36 @@
+title: Advection-diffusion of a Gaussian Hill
+@warning WORK IN PROGRESS @endwarning
+
+# Advection-diffusion of a Gaussian Hill # {#eg_GPP}
+
+In this example, we will simulate the time evolution of a Gaussian hill
+in an infinite space with periodic boundaries.
+The initial form of the Gaussian pulse gives the form:
+  $$ C(\mathbf{x}, t=0) = C_0 \exp\left(-\frac{(\mathbf{x}
+  - \mathbf{x}_0)^2}{2\sigma_0^2}\right) + C_{base} $$
+The analytical solution with time is:
+    $$ C(\mathbf{x}, t) = \frac{\sigma_0^2}{\sigma_0^2 + \sigma_D^2} C_0 \exp \left(
+    -\frac{(\mathbf{x} - \mathbf{x}_0)^2}{2(\sigma_0^2 + \sigma_D^2)} \right) + C_{base} $$
+
+In our simulation, $\sigma_0=40$ is used as a default value. A 1000*1000 2D mesh is recognized
+ here as an infinite area, and the D2Q9 layout is used. Boundaries from all directions are
+ periodic. As the area is much larger than the $ \sigma_0 $, the effect from the boundaries
+ is neglected. Results of t=200 are used to calculate the diffusion factor.
+
+![The comparison of concentration between simulation and analytical solution](media/compare_sim_ana_C.jpg)
+![The error of diffusion factor when u\_bg=0](media/err_numerical_diffusion.jpg)
+![Absolute error of D with u\_bg and order of equilibriums](media/d_abs_err_tau0.501.jpg)
+![Absolute error of D with u\_bg and order of equilibriums](media/d_abs_err_tau0.503.jpg)
+![Absolute error of D with u\_bg and order of equilibriums](media/d_abs_err_tau0.8.jpg)
+![Absolute error of D with u\_bg and order of equilibriums](media/d_abs_err_tau2.jpg)
+![Absolute error of D with u\_bg and order of equilibriums](media/d_abs_err_tau5.jpg)
+![The model error for first equilibrium](media/d_1st_model_error.jpg)
```

Comment:
Please add the script used to create the plots.

Reply:
The plot scripts have been added.